libcontainer/nsenter/nsexec.c

```diff
@@ -38,6 +38,9 @@ enum sync_t {
 	SYNC_ERR = 0xFF, /* Fatal error, no turning back. The error code follows. */
 };
 
+/* Synchronisation value for cgroup namespace setup */
+#define SYNC_PRIVATE_CGNS 0x80
```

Comment:
Since this synchronisation is different to the previous ones, we should consider naming it something distinctive. I actually didn't catch that this was a different type of synchronisation to the previous one on my first reading of this.